uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/import.jsp

```diff
@@ -115,41 +115,41 @@
                         <div class="flc-uploader-total-progress fl-uploader-total-progress-okay">&nbsp;</div>
                         <div class="flc-uploader-errorsPanel fl-uploader-errorsPanel">
                              <div class="fl-uploader-errorsPanel-header"><span class="flc-uploader-errorPanel-header">Warnings:</span></div>
-            
+
                              <!-- The markup for each error section will be rendered into these containers. -->
                              <div class="flc-uploader-errorPanel-section-fileSize"></div>
                              <div class="flc-uploader-errorPanel-section-numFiles"></div>
-                             
+
                              <!-- Error section template.-->
                              <div class="flc-uploader-errorPanel-section-tmplt fl-uploader-hidden-templates">
                                  <div class="flc-uploader-errorPanel-section-title fl-uploader-errorPanel-section-title">
                                      <p>x files were too y and were not added to the queue.</p>
                                  </div>
-                                 
+
                                  <div class="flc-uploader-errorPanel-section-details fl-uploader-errorPanel-section-details">
                                      <p>The following files were not added:</p>
                                      <p class="flc-uploader-errorPanel-section-files">file_1, file_2, file_3, file_4, file_5 </p>
                                  </div>
-                                 
+
                                  <button type="button" class="flc-uploader-errorPanel-section-toggleDetails fl-uploader-errorPanel-section-toggleDetails btn btn-default">Hide this list <i class="fa fa-eye-slash"></i></button>
                                  <button type="button" class="flc-uploader-errorPanel-section-removeButton fl-uploader-errorPanel-section-removeButton btn btn-warning">
                                      <span class="flc-uploader-erroredButton-text fl-uploader-hidden">Remove error <i class="fa fa-remove"></i></span>
                                  </button>
                              </div>
-                         </div> 
+                         </div>
                     </div>
                 </div>
-                
+
                 <!-- Action buttons -->
                 <div class="fl-uploader-btns btn-inline-group">
-                    <button type="button" class="flc-uploader-button-upload fl-uploader-upload fl-uploader-button-default fl-uploader-dim btn btn-success" disabled="disabled">Upload <i class="fa fa-upload"></i></button>
+                    <button type="submit" class="flc-uploader-button-upload fl-uploader-upload fl-uploader-button-default fl-uploader-dim btn btn-success" disabled="disabled">Upload <i class="fa fa-upload"></i></button>
```

Comment:
Could you by the way remove the hardcoded "Upload" and "Stop upload" messages in the two buttons, It can save time and PR comming from non US ?
for Messages.properties : 
```
import.button.submit.upload = Upload
import.button.submit.stop.upload = Stop Upload
```
for Messages_fr.properties : 
```
import.button.submit.upload = T\u00E9l\u00E9charger
import.button.submit.stop.upload = Arr\u00EAter le t\u00E9l\u00E9chargement
```